### PR TITLE
chore: use normal target directory with debug output

### PR DESCRIPTION
**Pull request by suztomo**

### CODE REVIEW

File: .kokoro/release/stage.sh [hunk 1]
```diff
@@ -39,11 +39,19 @@ mvn clean deploy -B \
...
   -P release
 
+echo "Finding nexus-staging"
+find . -name "nexus-staging"
+
+echo "Done"
+echo "Finding staging"
+find . -name "staging"
+echo  "Done"
+
```
Comment (suztomo) [inline .kokoro/release/stage.sh:49]: These lines clarify how the two mvn commands communicate staging repository IDs.

It didn't find any:

```
Finding nexus-staging
Done
Finding staging
Done
```

**Comment by suztomo** [comment]
Another attempt: http://fusion2/2d92ca3b-1b9d-4034-95dd-cd7c6cbd6094

File: gapic-generator-java/pom.xml [hunk 2]
```diff
@@ -54,7 +54,6 @@
       </resource>
     </resources>
     <testSourceDirectory>../src/test/java</testSourceDirectory>
-    <directory>../target</directory>
```
Comment (suztomo) [inline gapic-generator-java/pom.xml:57]: With this removal, now the first mvn command created the desired directory:

```
Finding nexus-staging
./gapic-generator-java/target/nexus-staging
Done
Finding staging
./gapic-generator-java/target/nexus-staging/staging
Done
AUTORELEASE_PR is not set. Not releasing.
```

**Comment by suztomo** [comment]
```
Error:  Errors: 
Error:    SourceCodeInfoParserTest.setUp:49->buildFileDescriptor:158->newDescriptorSetInputStream:181 » NoSuchFile test-proto.descriptorset
Error:    SourceCodeInfoParserTest.setUp:49->buildFileDescriptor:158->newDescriptorSetInputStream:181 » NoSuchFile test-proto.descriptorset
Error:    SourceCodeInfoParserTest.setUp:49->buildFileDescriptor:158->newDescriptorSetInputStream:181 » NoSuchFile test-proto.descriptorset
Error:    SourceCodeInfoParserTest.setUp:49->buildFileDescriptor:158->newDescriptorSetInputStream:181 » NoSuchFile test-proto.descriptorset
```

**Review by blakeli0 — APPROVED** [review]

**Comment by blakeli0** [comment]
Thanks for the troubleshooting @suztomo ! Not needed anymore.

**Pull request closed without merging**